Tensile/Source/client/source/SolutionIterator.cpp

```diff
@@ -141,7 +141,7 @@ namespace Tensile
         {
             m_reporter->report(ResultKey::SolutionIndex, m_currentSolutionIdx);
             m_reporter->report(ResultKey::SolutionProgress,
-                               concatenate(m_currentSolutionIdx, "/", m_lastSolutionIdx));
+                               concatenate(m_currentSolutionIdx, "/", m_lastSolutionIdx - m_firstSolutionIdx + 1));
```

Comment:
Won't this end up printing e.g. 12/10 if m_firstSolutionIdx > 0?

Reply:
Looks like possible. 
Somehow I just want to make the ProblemProgress (Idx/Size) consistent with SolutionProgress(Idx/LastIdx).
So I guess it's also possible for ProblemProgress to print 12/10 ?
How about unify them both to Idx/LastIdx ?